### ISSUE TITLE
Explain why new contributors can't save a post as draft

### ISSUE THREAD

**Issue by cesswairimu** [publiclab/plots2#9977]
A new contributor on PublicLab cannot create drafts and there's no help text or guide in the post page that helps the user understand this. I remember being really confused when I wanted to save my proposal as a draft.

<img width="1280" alt="Screenshot 2021-07-09 at 09 55 30" src="https://user-images.githubusercontent.com/63427719/125052372-ce434580-e09b-11eb-943b-7d335e26cdf3.png">

A small text could be added at the bottom that says `New contributors are not allowed to create drafts`.

_Originally posted by @TildaDares in https://github.com/publiclab/plots2/issues/9888#issuecomment-877031292_

**Comment by phyleria** [comment]
Hi @cesswairimu can I work on this one?

**Comment by cesswairimu** [comment]
Sure @phyleria, we would love your help. Thanks

**Comment by ebarry** [comment]
oh this is interesting, thanks for working on this!

Separately, @jywarren can we revisit our reasoning on whether new / unmoderated contributors should be able to save drafts?

**Comment by jywarren** [comment]
Sure - let's dig into the code to figure this out!

https://github.com/publiclab/plots2/blame/main/app/views/editor/rich.html.erb#L329-L343

So here is the corresponding pull request where that condition was added:

https://github.com/publiclab/plots2/pull/2747

Here's the planning issue by @grvsachdeva -- https://github.com/publiclab/plots2/issues/2667

Ah - here it is -- @steviepubliclab asked for it to be turned off for first-timers for a while to see how it goes, with the option to turn it on later:

https://github.com/publiclab/plots2/issues/2667#issuecomment-392165350

Would you like to revisit this decision now, or just notify people why? Thanks!

**Comment by ebarry** [comment]
I'm checking with Stevie _et al_ now. 
In the meantime, can i ask the audience, as there are several people here who have recently posted their first research note, who may have an opinion on whether the save as draft option would have been helpful or too overwhelming?

**Comment by steviepubliclab** [comment]
I'm good with whatever new users are finding works best.

**Comment by ebarry** [comment]
@cesswairimu and @TildaDares -- do you think first time posters should have access to the Save as Draft feature? @publiclab/community-reps are open to this change. Please advise. 

FYI, our original reasoning was that new posters would hesitate too long perfecting their draft when it's better to just get their post out there. That would mean two things: 

- that the small group of moderators (mostly staff) would be required to go through a growing queue of draft posts to individually answer how-to questions in a space that wasn't visible to the entire community. 
- that draft posts would be flooded with spam and we'd have to route not only submitted spam but draft spam to a spam queue. This is perhaps the killer point to this idea -- this would be an unworkable situation.

**Comment by TildaDares** [comment]
@ebarry I’m okay with whichever one but there should be a way to inform new contributors that they can’t save drafts if you decide to stick to the current arrangement.

**Comment by ebarry** [comment]
great, i agree, thanks @TildaDares . 

I'm wondering—is the "Save as Draft" button actually visible to first-time posters? I'm asking because i do not see it in your screenshot.  Perhaps were you expecting this button to be present because you know that this feature exists, and had seen the feature previously on stable.publiclab.org?

**Comment by cesswairimu** [comment]
@ebarry save as draft could be beneficial to soc applicants when drafting their proposals but  from what I am hearing, more work would go to moderators on managing draft spams...I would opt for leaving it as is for that reason.  thanks

**Comment by TildaDares** [comment]
@ebarry No, the `Save as Draft` is not visible to first time contributors. When I was drafting my proposal I expected it to be there because I had seen it in my local environment.

**Comment by ebarry** [comment]
aha. perfect. So, let's craft the message that's more like: "After you publish your first post, and it is approved by human moderators, you will have access to a 'Save as Draft' feature that you can use for creating future content."

This could be improved! Do you see what i'm getting at? As in, don't just tell people there's a feature they've never heard of that they don't have access to.

**Comment by ebarry** [comment]
Does this PR resolve this issue? 
Thinking about this being project scoping season, it would be great to go ahead and commit the work we've done previously to streamline onboarding.

**Comment by cesswairimu** [comment]
From the PR , it looks like it will resolve it, I will follow up to have it merged. Thanks

**Comment by cesswairimu** [comment]
@shefali12-ab this is a good one.  Would you like to work on it?

**Comment by shefali12-ab** [comment]
@cesswairimu I just need to add small text saying new contributors are not allowed to create draft ?

**Comment by cesswairimu** [comment]
yeah, maybe lets convert this to an fto. Take a look of this instead https://github.com/publiclab/mapknitter/issues/1634

**Comment by shefali12-ab** [comment]
Sure @cesswairimu I will look into that issue, this week I'm having my exams , I will get back to https://github.com/publiclab/mapknitter/issues/1634 issue soon. Thank you :)
